core/contract/src/main/java/org/eclipse/dataspaceconnector/contract/negotiation/ConsumerContractNegotiationManagerImpl.java

```diff
@@ -255,6 +259,7 @@ public NegotiationResult declined(ClaimToken token, String negotiationId) {
      * @param process The contract negotiation.
      * @return The response to the sent message.
      */
+    @WithSpan(value = "sending negotiation offer")
```

Comment:
See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span for advice on span names